src/i_sound.c

```diff
@@ -584,9 +412,8 @@ void I_InitSound(void)
         if (!S_sfx[i].name)
           continue;
 
-        CacheSound(&S_sfx[i], 0);
+        sound_module->CacheSound(&S_sfx[i]);
       }
-      StopChannel(0);
```

Comment:
Would it make sense for this entire section to be in `i_oalsound.c`? Again, I struggle to picture how hot swapping to PC speaker sound will affect all of this.

Reply:
We'll just cache the sounds on the fly if we switch from the PC Speaker. In the case of OpenAL, pre-caching is not so important.